extensions/wikia/AssetsManager/config.php

```diff
@@ -2211,14 +2211,20 @@
 	],
 ];
 
-$config['on_site_notifications_js'] = [
+$config['design_system_on_site_notifications_js'] = [
 	'type' => AssetsManager::TYPE_JS,
 	'skin' => [ 'oasis' ],
 	'assets' => [
 		'//extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.js'
 	],
 ];
 
+$config['design_system_on_site_notifications_mustache_js'] = [
+	'type' => AssetsManager::TYPE_JS,
+	'skin' => [ 'oasis' ],
+	'assets' => [],
```

Comment:
If you're going to implement this incrementally IMO it would be better if you skipped the placeholders "for the future". I guess you will start using it at some point but if I don't know for sure, then it's hard to do a proper CR.

It makes me want to say "remove it, it's not being used" :wink: